runners.go

```diff
@@ -324,3 +326,90 @@ func (s *RunnersService) DisableProjectRunner(pid interface{}, rid interface{},
 
 	return s.client.Do(req, nil)
 }
+
+// NewRunner represents the GitLab CI runner registration response.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/runners.html#register-a-new-runner
+type NewRunner struct {
+	ID    string `url:"id" json:"id"`
+	Token string `url:"token" json:"token"`
+}
```

Comment:
I think I prefer to reuse the `Runner` struct for this one. We can do that by just adding the `token` field to that struct. Does that make sense?

Reply:
I totally agree. I just got disturbed by the doc which is misleading: it implies the `id` is a string which actually isn't, it's an int. So the Runner struct is appropriate. Thanks for the review!